audio/out/ao_pipewire.c

```diff
@@ -185,7 +185,10 @@ static void on_param_changed(void *userdata, uint32_t id, const struct spa_pod *
         return;
     }
 
-    pw_stream_update_params(p->stream, params, 1);
+    if (pw_stream_update_params(p->stream, params, 1) < 0) {
+        MP_ERR(ao, "Could not update stream parameters\n");
+        return;
```

Comment:
No need `return`. It's already the end of the function.

Reply:
This was intentional to be consistent between the different error paths and make it explicit that we want to return even if we later extend this function.

If you prefer to remove it I can remove it though.

Reply:
Fair enough. let's leave as it then.